draft-ietf-quic-multipath.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Multipath Extension for QUIC
+title: Managing multiple paths for a QUIC connection
 abbrev: Multipath QUIC
```

Comment:
Seems we should update the abbrev too? We never wanted to call this "multipath QUIC" even with the old title

Reply:
`Multiple paths for QUIC` ?

Reply:
Changed to "Multiple paths for QUIC"

Reply:
But the docs doesn't specify "Simultaneous Data Transmission". That's my main point.